package.json

```diff
@@ -12,11 +12,13 @@
     "lint:ci": "eslint . --ext .ts",
     "lint:fix": "eslint . --ext .ts --fix",
     "build": "npm-run-all clean tsc copy-assets",
-    "pretest": "docker compose up -d db-test",
+    "pretest-docker": "docker compose up -d db-test",
+    "pretest-podman": "podman compose up -d db-test",
     "test": "NODE_ENV=test jest --coverage",
     "test:ci": "NODE_ENV=test jest --ci --coverage --config=jest.config.ts",
     "check": "npm-run-all prettier:fix lint:fix test build",
-    "predev": "docker compose up -d db-dev",
+    "predev-docker": "docker compose up -d db-dev",
+    "predev-podman": "podman compose up -d db-dev",
```

Comment:
This is going to break the current script functionality - npm will automatically run `preX` scripts when you `npm run X`, so `npm run test` and `npm run dev` will stop automatically bringing up the relevant containers.

https://docs.npmjs.com/cli/v10/using-npm/scripts#pre--post-scripts

Maybe a quick alternative here is to add a docker alias for podman in your `.zshrc`?

```
alias docker='podman'
```

That should allow the pre scripts to work as intended.

Reply:
"Should!"... Actually yeah we can probably do that... The issue was the mounts in the docker-compse tripping up podman.  I'll revert that :-)